core/test/ManualPriceFeed.js

```diff
@@ -11,7 +11,7 @@ contract("ManualPriceFeed", function(accounts) {
   let rando = accounts[1];
 
   before(async function() {
-    manualPriceFeed = await ManualPriceFeed.deployed();
+    manualPriceFeed = await ManualPriceFeed.new(true);
```

Comment:
Is the migration coming in a later PR?

Reply:
Yeah, mainly because the numbers get all screwed up when Yutaro and Roz merge their migrations in.